tests/foreman/api/test_smartproxy.py

```diff
@@ -41,9 +42,10 @@ def setUpClass(cls):
         existing smart proxy should always succeed.
         """
         super(SmartProxyMissingAttrTestCase, cls).setUpClass()
-        smart_proxies = entities.SmartProxy().search()
-        assert len(smart_proxies) > 0
```

Comment:
If i'm not wrong, the only way to mark test as failed is to fail unittest's assertion. Default python assertion will mark class as errored and we can't use unittest here, so class will be marked as 'error' regardless. The only question here is whether  `list index out of range` message is as clear as `0 != 1` message.